Assets/Scripts/System/FileAssetLoaderSystem.cs

```diff
@@ -0,0 +1,388 @@
+using Assets.Scripts.EditorState;
```

Comment:
We need a test for the FileAssetLoaderSystem
There is too much, that can go wrong with this

Also, I'd recommend rebasing everything on the current Refactoring branch. There is a MockPathState which can come in handy.